config-schema.json

```diff
@@ -404,7 +404,13 @@
                       ],
                       "title": "IR code format",
                       "default": null
-                    }
+                    },
+                    "device": {
+                      "$id": "#/properties/entities/properties/remote/items/properties/commands/items/properties/device",
+                      "type": "string",
+                      "default": null,
+                      "pattern": ""
+                    },
```

Comment:
This is invalid json: most likely the comma.

    error:  /home/runner/work/remote-software/remote-software/config-schema.json: Unexpected token } in JSON at position 16059